share/spice/magic_the_gathering/magic_the_gathering.js

```diff
@@ -1,45 +1,75 @@
 (function (env) {
     "use strict";
+
+    /**
+     * Get the search term from the DDG query that triggered this IA.
+     *
+     * @return {String} The unencoded search term without the IA trigger words.
+     */
+    function getSearchTerm() {
+        var query = DDG.get_query();
+        var triggers = [
+            'magic card',
+            'mtg',
+            'magic: the gathering',
+            'magic the gathering'
+        ];
+        return query.replace(new RegExp(triggers.join('|')), '').trim();
+    }
```

Comment:
Seeing as the none of this is going to change, I'd say this should be stored in a variable instead of a function:

```suggestion
var query = DDG.get_query(),
    triggerRe = /mtg|magic:? (card|the gathering)/ig,
    searchTerm = query.replace(triggerRe, '').trim();
```

You can then use query below to replace subsequent calls to DDG.get_query();

Reply:
@moollaza Yeah that works.

Do I apply this suggestion, and then make a commit updating `searchTerm()` calls in the code, or do I do all changes in one commit? I'm not sure if doing the latter will remove that you are the author of the above suggestion in the commits.